config/initializers/name_generator_seed.rb

```diff
@@ -0,0 +1,6 @@
+if (Setting['name_generator_seed'] && Setting['name_generator_seed'] < 1) || (Setting['name_generator_register'] && Setting['name_generator_register'] < 1)
```

Comment:
I'd say let's just ship a seed and register and forget about letting users change it